.github/workflows/ci.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
             runalyzer-gather summary.sqlite log.sqlite
             runalyzer -m summary.sqlite -c 'dbplot(q("select $t_sim, $t_step"))'
 
+            export OMPI_MCA_rmaps_base_oversubscribe=1
```

Comment:
You say "on MacOS", but this change doesn't seem limited to MacOS.

Reply:
Ok, fair. It's an OpenMPI change, that only affects MacOS in CI since Linux uses mpich ;-)